plugins/modules/accesspoint_workflow_manager.py

```diff
@@ -0,0 +1,818 @@
+                del device["apNameNew"]
+            elif device.get("apName") is None and device.get("macAddress") is not None:
+                device["apList"] = [dict(macAddress = device["macAddress"])]
+
+            if device.get("location") is not None:
+                device["configureLocation"] = True
+            if device.get("ledBrightnessLevel") is not None:
+                device["configureLedBrightnessLevel"] = True
+            if device.get("ledStatus") is not None:
+                device["configureLedStatus"] = True
+                device["ledStatus"] = True if device["ledStatus"] == "Enabled" else False
+
+            for rmkey in ("mac_address", "hostname", "management_ip_address"):
+                if device.get(rmkey): del device[rmkey]
+            
+            #response = self.dnacsdk.wireless.configure_access_points(**device)
+            response = self.dnac._exec(
+                    family="wireless",
+                    function='configure_access_points',
+                    op_modifies=True,
+                    params=device,
+                )
+
+            self.log("Response of Access Point Configuration: " + str(response["response"]), "INFO")
+            return dict(macAdress=device["macAddress"], response=response["response"])
+
+        except Exception as e:
+            self.log("AP config update Error" + device["macAddress"] + str(e), "ERROR")
+
+    def data_frame(self, fieldlist = None, data = list):
+        """
+        This function used to give output as limited field to show for given in the fieldlist
+        need to pass the list of field need to show with , separated string value also need
+        to pass the data as device list or device config list as input
+        Parameters:
+          - fieldlist: (str) key need to display the value.
+            eg : device_fields: "id,hostname,family,type,mac_address,up_time"
+          - data: (list of dict) Device list or Config list
+        Returns:
+            {
+                "family": "Unified AP",
+                "hostname": "LTTS-Test2",
+                "id": "34f5a410-413d-4a6c-b195-8267fd599491",
+                "last_updated": "2024-06-05 13:06:24",
+                "mac_address": "34:5d:a8:3b:d8:e0",
+                "up_time": "7 days, 13:07:00.020"
+            }
+        Example:
+            functions = Accesspoint(module)
+            final_input_data = functions.data_frame(device_fields, device_data)
+        """
+        try:
+            data = self.camel_to_snake_case(data)
+            if len(data) != 0 and fieldlist is not None:
+                fields = [x for x in fieldlist.split(",")]
+                dataframe = []
+                for eachdata in data:
+                    limitedfields = {}
+                    for each_key in fields:
+                        limitedfields[each_key] = eachdata[each_key]
+                    dataframe.append(limitedfields)
+                return dataframe
+            else:
```

Comment:
remove else block

Reply:
Which need to return either same data or filtered data.